<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="FilterLFOGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>232</width>
      <height>156</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Fitler LFO</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="FilterLFORateHorizontalLayout">
       <item>
        <widget class="QDial" name="FilterLFORateKnob">
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="notchTarget">
          <double>10.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="FilterLFORateKnobLabel">
         <property name="text">
          <string>Rate</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="FilterLfoButtonsHorizontalLayout">
       <item>
        <widget class="QRadioButton" name="FilterLFOSinButton">
         <property name="text">
          <string>sin</string>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="FilterLFOSawButton">
         <property name="text">
          <string>saw</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="FilterLFOSqrButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>sqr</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="FilterLFOTriButton">
         <property name="text">
          <string>tri</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="FilterLFOHorizontalLayout">
       <item>
        <widget class="QDial" name="FilterLFOAmountKnob">
         <property name="notchTarget">
          <double>10.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="FilterLFOAmountLabel">
         <property name="text">
          <string>Amount</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Oscilators</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_21">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="Osc1GroupBox">
          <property name="title">
           <string>Osc1</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QVBoxLayout" name="Osc1verticalLayout">
             <item>
              <widget class="QRadioButton" name="RadioButton_Osc1_Sinus">
               <property name="text">
                <string>sin</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>10</width>
                 <height>10</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioButton_Osc1_Saw">
               <property name="text">
                <string>saw</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioButton_Osc1_Square">
               <property name="text">
                <string>sqr</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioButton_Osc1_Triangle">
               <property name="text">
                <string>tri</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="Osc1GroupBox_2">
          <property name="title">
           <string>Osc2</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_19">
           <item>
            <layout class="QVBoxLayout" name="Osc1verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="RadioButton_Osc2_Sinus">
               <property name="text">
                <string>sin</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>10</width>
                 <height>10</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioButton_Osc2_Saw">
               <property name="text">
                <string>saw</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioButton_Osc2_Square">
               <property name="text">
                <string>sqr</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="RadioButton_Osc2_Triangle">
               <property name="text">
                <string>tri</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="MixGroupBox">
       <property name="title">
        <string>Mix</string>
       </property>
       <widget class="QSlider" name="MixSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="AmplifierGroupBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Amplifier</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="VolumeAttackHorizontalLayout">
       <item>
        <widget class="QDial" name="VolumeAttackKnob">
         <property name="notchTarget">
          <double>10.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="VolumeAttackKnobLabel">
         <property name="text">
          <string>Attack</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VolumeDecayHorizontalLayout">
       <item>
        <widget class="QDial" name="VolumeDecayKnob">
         <property name="notchTarget">
          <double>10.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="VolumeDecayKnobLabel">
         <property name="text">
          <string>Decay</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VolumeSustainHorizontalLayout">
       <item>
        <widget class="QDial" name="VolumeSustsainKnob">
         <property name="notchTarget">
          <double>10.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="VolumeSustsainKnobLabel">
         <property name="text">
          <string>Sustain</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VolumeReleaseHorizontalLayout">
       <item>
        <widget class="QDial" name="VolumeReleaseKnob">
         <property name="notchTarget">
          <double>10.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="VolumeReleaseKnobLabel">
         <property name="text">
          <string>Release</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VolumeGainHorizontalLayout">
       <item>
        <widget class="QDial" name="VolumeGainKnob">
         <property name="notchTarget">
          <double>10.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="VolumeGainKnobLabel">
         <property name="text">
          <string>Gain</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="FilterMainGroupBoxHorizontalLayout">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>412</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="FilterGroupBoxVerticalLayout">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="FilterTypeButtonsHorizontalLayout">
          <item>
           <widget class="QRadioButton" name="BpButton">
            <property name="text">
             <string>BP</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="HpButton">
            <property name="text">
             <string>HP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="LpButton">
            <property name="text">
             <string>LP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FilterTypeLabel">
            <property name="text">
             <string>Filtertype</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="FilterEnvelopeAttackHorizontalLayout">
          <item>
           <widget class="QDial" name="FilterEnvelopeAttackKnob">
            <property name="notchTarget">
             <double>10.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FilterEnvelopeAttackKnobLabel">
            <property name="text">
             <string>Attack</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="FilterDecayHorizontalLayout">
          <item>
           <widget class="QDial" name="FilterDecayKnob">
            <property name="notchTarget">
             <double>10.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FilterDecayKnobLabel">
            <property name="text">
             <string>Decay</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="FilterSustainHorizontalLayout">
          <item>
           <widget class="QDial" name="FilterSustainKnob">
            <property name="notchTarget">
             <double>10.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FilterSustainKnobLabel">
            <property name="text">
             <string>Sustain</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="FilterReleaseHorizontalLayout">
          <item>
           <widget class="QDial" name="FilterReleaseKnob">
            <property name="notchTarget">
             <double>10.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FilterReleaseKnobLabel">
            <property name="text">
             <string>Release</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="FilterSecondRowVerticalLayout">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="CutOffHorizontalLayout">
          <item>
           <widget class="QDial" name="CutOffKnob">
            <property name="notchTarget">
             <double>10.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="CutOffLabel">
            <property name="text">
             <string>Cutoff</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ResonanceHorizontalLayout">
          <item>
           <widget class="QDial" name="ResonanceKnob">
            <property name="notchTarget">
             <double>10.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ResonanceKnobLabel">
            <property name="text">
             <string>Resonance</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="EnvAmntHorizontalLayout">
          <item>
           <widget class="QDial" name="dial_17">
            <property name="notchTarget">
             <double>10.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Env amnt</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Button_Octave_UP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Octave UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Octave_Down">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Octave Down</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
